roles/edpm_ssh_known_hosts/tasks/main.yml

```diff
@@ -73,6 +73,13 @@
           {%       set _ = entries.append(hostdata['canonical_hostname']) %}
           {%     endif %}
           {%     set _ = entries.append(host ~ '*') %}
+          {%     if 'ansible_port' in hostdata and hostdata['ansible_port'] != 22 %}
```

Comment:
this needs to be `hostdata['ansible_port']|int` to get make the int comparison work